backend/pom.xml

```diff
@@ -18,6 +18,7 @@
 		<kotlin.version>1.6.21</kotlin.version>
 	</properties>
 	<dependencies>
+		<!-- https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp -->
```

Comment:
I think we can remove this if we don't and won't use it

Reply:
Done